src/Raven.Server/Documents/DatabaseRequestHandler.cs

```diff
@@ -157,5 +165,68 @@ protected void AddPagingPerformanceHint(PagingOperationType operation, string ac
+                case "etl-add":
+                    var etlType = EtlConfiguration<ConnectionString>.GetEtlType(configuration).ToString();
+                    if (etlType == "Raven")
+                        return JsonDeserializationClient.RavenEtlConfiguration(configuration).ToJson();
+                    if (etlType == "ElasticSearch")
+                        return JsonDeserializationClient.ElasticSearchEtlConfiguration(configuration).ToJson();
+                    if (etlType == "Queue")
+                        return JsonDeserializationClient.QueueEtlConfiguration(configuration).ToJson();
+                    if (etlType == "Sql")
+                        return JsonDeserializationClient.SqlEtlConfiguration(configuration).ToJson();
+                    if (etlType == "Olap")
+                        return JsonDeserializationClient.OlapEtlConfiguration(configuration).ToJson();
+                    break;
+                case "update_external_replication":
+                    return JsonDeserializationClient.ExternalReplication(configuration).ToJson();
+                case "update-hub-pull-replication":
+                    return JsonDeserializationClient.PullReplicationDefinition(configuration).ToAuditJson();
+                case "update-sink-pull-replication":
+                    return JsonDeserializationClient.PullReplicationAsSink(configuration).ToAuditJson();
+                case "put-connection-string":
+                    var connectionStringType = ConnectionString.GetConnectionStringType(configuration).ToString();
+                    if (connectionStringType == "Raven")
+                        return JsonDeserializationClient.RavenConnectionString(configuration).ToJson();
+                    if (connectionStringType == "ElasticSearch")
+                        return JsonDeserializationClient.ElasticSearchConnectionString(configuration).ToAuditJson();
+                    if (connectionStringType == "Queue")
+                        return JsonDeserializationClient.QueueConnectionString(configuration).ToJson();
+                    if (connectionStringType == "Sql")
+                        return JsonDeserializationClient.SqlConnectionString(configuration).ToJson();
+                    if (connectionStringType == "Olap")
+                        return JsonDeserializationClient.OlapConnectionString(configuration).ToJson();
+                    break;
+            }
+            return null;
+        }
+
+        protected void LogTaskToAudit(string description, string id, DynamicJsonValue conf)
+        {
+            if (LoggingSource.AuditLog.IsInfoEnabled)
+            {
+                var clientCert = GetCurrentCertificate();
+                var auditLog = LoggingSource.AuditLog.GetLogger(Database.Name, "Audit");
+                var line = $"Task: {description} with taskId: {id} executed by {clientCert?.Subject} {clientCert?.Thumbprint} ";
```

Comment:
```suggestion
                var line = $"Task: '{description}' with taskId: '{id}' executed by {clientCert?.Subject} {clientCert?.Thumbprint} ";
```

also if clientCert is null, please make simple message (without executed by).

Reply:
Done